### DISCUSSION TITLE
return result from operator A pass to B

### DISCUSSION THREAD

**Discussion by reoxu** [apache/airflow#27568] [Q&A] [answered]
I'm trying to pass return value from a operator to the following operator.
In bash operator it seems working, while in my custom operator it doesn't working.How can i do this in right way.
Here is my code:    
![image](https://user-images.githubusercontent.com/114573733/200799139-42941b1a-35ff-4ed7-a151-869bb20fbad1.png)
![image](https://user-images.githubusercontent.com/114573733/200799280-4850d726-e178-43eb-afde-570df1468d9a.png)
![image](https://user-images.githubusercontent.com/114573733/200799359-8164b841-c9f3-4b53-aba0-7ac9e9bada98.png)
![image](https://user-images.githubusercontent.com/114573733/200799591-75e06a86-12d5-4563-85ff-c259f4d36564.png)
![image](https://user-images.githubusercontent.com/114573733/200799485-e83c8d10-76ef-43a1-9538-e19cb3cfcdf6.png)

**Comment by Taragolis** [comment, marked as answer]
@reoxu If I've got you correctly your `JobInstanceStateSensor` couldn't render templates.
The reason - You forgot to specify class attribute `template_fields`: https://airflow.apache.org/docs/apache-airflow/stable/howto/custom-operator.html#templating

In additional (not related to your issue) better to paste code/logs as a text rather than screenshots. Github supports [markdown syntax](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks) for that

````
```python
from your.awesome import module

1 / 0
```

```
Just a text
```
````

**Reply by reoxu** [reply]
Thanks for your help, after adding the template_fields it's working.
In addition, I will paste my code as a text in the future.
